### ISSUE TITLE
Any plan to add Noam scheduling?

### ISSUE THREAD

**Issue by kwonmha** [pytorch/pytorch#78253]
### 🚀 The feature, motivation and pitch

I found Noam scheduling(from "Attention is all you need" paper) important for training transformers.
But it's not included in torch.
Do you have plan to add it or want others to add it?

**Comment by t-vi** [comment]
I think the idea is that you could use/subclass LambdaLR with ease.

**Comment by kwonmha** [comment]
Right. I can implement it with LambdaLR or adopt other implementations.

I think it's about policy or plan to include Noam scheme in this project.
So opened this issue.
And you mean it doesn't look good to provide that scheme in pytorch?
